src/rpc_client/src/meta_client.rs

```diff
@@ -457,6 +457,14 @@ impl MetaClient {
         Ok(resp.version)
     }
 
+    pub async fn alter_source_with_sr(&self, source: PbSource) -> Result<CatalogVersion> {
+        let request = AlterSourceRequest {
+            source: Some(source),
+        };
+        let resp = self.inner.alter_source(request).await?;
```

Comment:
Wow, the logics in the Meta node side are all completed before?

Reply:
Yes indeed. Since `alter source sr with table` is not implemented in this pr, we only need to replace PbSource in the Meta. The replacement is done by `alter_source` handler and thus can be reused.